COMPONENTS

```diff
@@ -1,23 +1,23 @@
 --------------------------------------------------------
 COMPONENT   | GIT BRANCH/TAG    | INSTALL AS
 --------------------------------------------------------
-bacio       | v2.4.0            | 2.4.0
-sigio       | v2.3.0            | 2.3.0
-sfcio       | v1.4.0            | 1.4.0
-gfsio       | v1.4.0            | 1.4.0
-nemsio      | v2.5.1            | 2.5.1
-nemsiogfs   | v2.5.0            | 2.5.0
-w3emc       | v2.7.0            | 2.7.0
-w3nco       | v2.4.0            | 2.4.0
-ip          | v3.3.0            | 3.3.0
-ip2         | v1.1.0            | 1.1.0
-sp          | v2.3.0            | 2.3.0
-landsfcutil | v2.4.0            | 2.4.0
-g2          | v3.4.0            | 3.4.0
-g2tmpl      | v1.9.0            | 1.9.0
+bacio       | v2.4.1            | 2.4.1
+sigio       | v2.3.2            | 2.3.2
+sfcio       | v1.4.1            | 1.4.1
+gfsio       | v1.4.1            | 1.4.1
+nemsio      | v2.5.2            | 2.5.2
+nemsiogfs   | v2.5.1            | 2.5.1
+w3emc       | v2.7.1            | 2.7.1
+w3nco       | v2.4.1            | 2.4.1
+ip          | v3.3.1            | 3.3.1
+ip2         | v1.1.1            | 1.1.1
+sp          | v2.3.1            | 2.3.1
+landsfcutil | v2.4.1            | 2.4.1
+g2          | v3.4.1            | 3.4.1
+g2tmpl      | v1.9.1            | 1.9.1
 crtm        | v2.3.0            | 2.3.0
 nceppost    | dceca26           | dceca26
-wrf_io      | v1.1.1-cmake      | 1.1.1
+wrf_io      | v1.1.1-cmake-v2      | 1.1.1
 bufr        | bufr_v11.3.1.1    | 11.3.1.1
 wgrib2      | v2.0.8-cmake-v1.0.3 | 2.0.8
---------------------------------------------------------
+--------------------------------------------------------
```

Comment:
Is the newline required in line 23? Just checking.

Reply:
no. it is just a text file and we `grep` for package names.  It is purely decorative.